src/ParaTest/Runners/PHPUnit/TestMethod.php

```diff
@@ -20,17 +20,16 @@ class TestMethod extends ExecutableTest
     protected $path;
 
     /**
-     * The name of the method that will be supplied
-     * to the --filter switch
+     * A set of filters for test, they are merged into phpunit's --filter option
      *
-     * @var string
+     * @var string[]
      */
-    protected $name;
+    protected $filters;
 
-    public function __construct($suitePath, $name)
+    public function __construct($testPath, $filters)
     {
-        $this->path = $suitePath;
-        $this->name = $name;
+        $this->path = $testPath;
+        $this->filters = (array)$filters;
```

Comment:
for compatibility with other code (tests), which can pass string (one filter) instead of array.